src/CraigslistAPI/CraigslistPostDetailsComponent.js

```diff
@@ -25,7 +26,15 @@ export default class CraigslistPostDetailsComponent extends React.Component {
                 <h2 className="mt-3">{this.state.listing.title}</h2>
                 <h4>{this.state.listing.price}</h4>
                 <div className="row">
-                    <div className="col"><p>{this.state.listing.description}</p></div>
+                    <div className="col">
+                        <p>{this.state.listing.description}</p>
```

Comment:
It seems like all the listing descriptions are prepended with "QR Link To This Post" before the actual description. I'm assuming that's just the value that is `this.state.listing.description` What do you think is the best solution/work-around? We could just display the latter part of the description, omitting the "QR Link To This Post"

Reply:
Resolved.